python_modules/dagster/dagster/_core/storage/upath_io_manager.py

```diff
@@ -370,6 +370,7 @@ async def collect():
                         context.log.error(str(result))
                         found_errors = True
                 elif isinstance(result, Exception):
+                    raise result
```

Comment:
intentional?

Reply:
artifact of how i got the stack trace, thanks for catching